tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/ConfidentialClientIntegrationTests.cs

```diff
@@ -390,7 +390,7 @@ private async Task RunOnBehalfOfTestAsync(LabResponse labResponse)
 
             SecureString securePassword = new NetworkCredential("", user.GetOrFetchPassword()).SecurePassword;
 
-            var msalPublicClient = PublicClientApplicationBuilder.Create(publicClientID).WithAuthority(TestConstants.AuthorityOrganizationsTenant).WithRedirectUri("urn:ietf:wg:oauth:2.0:oob").Build();
+            var msalPublicClient = PublicClientApplicationBuilder.Create(publicClientID).WithAuthority(TestConstants.AuthorityOrganizationsTenant).WithRedirectUri("https://login.microsoftonline.com/common/oauth2/nativeclient").Build();
```

Comment:
I'm surprised there is not a const available to be used here given the number of places you have been updating consts

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/9b6ed7aa6fba9f7063e1e72a2a1a609f620f1332/src/client/Microsoft.Identity.Client/Core/Constants.cs#L16